.github/workflows/ci-latest-helm-chart-release.yaml

```diff
@@ -22,10 +22,16 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Update Chart.yaml
-        run: |
+        run: |  
+
           STABLE_VERSION=`cat STABLE-RELEASE`
-          sed -i -e "s/appVersion:.*/appVersion: $STABLE_VERSION/g" deployments/helm/*/Chart.yaml
-          sed -i -e "s/version:.*/version: $STABLE_VERSION/g" deployments/helm/*/Chart.yaml
+          for dep in deployments/helm/*/Chart.yaml; do
+            OLD_CHART_VERSION=`cat $dep | grep version | cut -d " " -f 2`
+            NEW_CHART_VERSION=`echo $OLD_CHART_VERSION | awk -F. '{$NF=$NF+1; print}' | tr ' ' '.'`
```

Comment:
This only handles patch releases, we also need to think about the case where we will need a minor/major release. 
Maybe we can use some other approach here.